Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/chef/omnibus-software.git
-  revision: bce0974a4f64bce2cc100a5baae26031ea430611
+  revision: 282c422ddb8b8e4af9d2f1cb8011a280e63f5308
```

Comment:
Just wondering, do we need chef/omnibus-software#1453 as well (commit [`a901ec26f8c704ba793c0609319f00f49e630d6b`](https://github.com/chef/omnibus-software/commit/a901ec26f8c704ba793c0609319f00f49e630d6b))?

Reply:
Done!